.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
-before_install: gem install bundler -v '1.17.3'
-install: bundle _1.17.3_ install --jobs=3 --retry=3
+before_install: gem install bundler -v '1.17.2'
+install: bundle _1.17.2_ install --jobs=2 --retry=2 --path=${BUNDLE_PATH:-vendor/bundle}
```

Comment:
Why did we go from bundler 1.17.3 to 1.17.2?

Reply:
This was required to get the Ruby 2.6 builds to work - I think because Ruby comes with it bundled in now. It was throwing a mismatched versions error.

Reply:
weird, the `before_install` script should have forced installation of the newest version and then the `install` script should have used that specific version.  In any event, it's not a major difference and I don't think it impacts our usage.

Reply:
Agreed. In any case, once we can drop Rails 4.2 support we can get rid of this line entirely and it will automatically use Bundler 2.0.